roles/install_nextcloud/tasks/tls_signed.yml

```diff
@@ -36,18 +36,19 @@
     src: "{{ nextcloud_tls_src_chain }}"
     owner: "{{ nextcloud_websrv_user }}"
     group: "{{ nextcloud_websrv_group }}"
-    mode: 0640
+    mode: "0640"
     force: false
   when: nextcloud_tls_src_chain is defined
 
-- name: "[SIGNED TLS] - Create chained certificate file for nginx"
-  ansible.builtin.assemble:
-    src: "{{ cert_path }}"
-    regexp: "{{ nextcloud_instance_name }}\\.crt|{{ nextcloud_instance_name }}\\.pem"
+- name: "[SIGNED TLS] - Overwrite and create chained certificate file for nginx"
```

Comment:
This seems to break idempotence as the file was also copied a few tasks earlier on the remote host.

Reply:
Thanks @aalaesar. My new commit shall fix this. The changes are minor, but I did re-arrange the tasks, which shall improve reading the code.